x-pack/plugins/observability_solution/infra/public/components/asset_details/template/page.tsx

```diff
@@ -79,7 +84,7 @@ export const Page = ({ tabs = [], links = [] }: ContentTemplateProps) => {
   return (
     <InfraPageTemplate
       onboardingFlow={asset.type === 'host' ? OnboardingFlow.Hosts : OnboardingFlow.Infra}
-      dataAvailabilityModules={asset.type === 'host' ? [SYSTEM_INTEGRATION] : undefined}
+      dataAvailabilityModules={DATA_AVAILABILITY_PER_TYPE[asset.type] || undefined}
```

Comment:
if the asset.type is not defined in DATA_AVAILABILITY_PER_TYPE it will be undefined.
```suggestion
      dataAvailabilityModules={DATA_AVAILABILITY_PER_TYPE[asset.type]}
```